### ISSUE TITLE
Hi, is there a kind of event planning/rsvp tool similar to Facebook's present in matrix? If there were, it would be much easier to make the switch. 

### ISSUE THREAD

**Issue by wvhulle** [matrix-org/matrix-spec#1006]
**Suggestion**
What would you like to see in Matrix? If your idea is vaguely complete enough, we
recommend submitting [an MSC](https://matrix.org/docs/spec/proposals) instead.

**Comment by ara4n** [comment]
Yeah, calendaring over Matrix (which is basically what this is) is something we'd like to do, as per https://github.com/matrix-org/matrix-spec/issues/284.

However, there's not an MSC yet.
